docs/SAM_VOICE_LIVEKIT.md

```
@@ -0,0 +1,94 @@
+# SAM Realtime Voice over LiveKit
+
+Talk to SAM hands-free and hear SAM talk back — full-duplex voice command and
+control of the agency, modelled on
+[keithschacht/taskmaster](https://github.com/keithschacht/taskmaster).
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)